jobs/build/ocp/Jenkinsfile

```diff
@@ -92,14 +105,20 @@ ${OA_CHANGELOG}
 """);
 
     try {
+            PARTIAL = " "
+            product_suffix = ""
+            if ( BUILD_EXCLUSIONS != "" ) {
+              PARTIAL = " PARTIAL "
+              product_suffix = " - PARTIAL"
+            }
             timeout(3) {
-                sendCIMessage( messageContent: "New build for OpenShift ${target}: ${version}",
+                sendCIMessage( messageContent: "New${PARTIAL}build for OpenShift ${target}: ${version}",
                         messageProperties: """build_mode=${BUILD_MODE}
                         puddle_url=${mirrorURL}/${OCP_PUDDLE}
                         image_registry_root=registry.reg-aws.openshift.com:443
                         brew_task_url_openshift=${OSE_BREW_URL}
                         brew_task_url_openshift_ansible=${OA_BREW_URL}
-                        product=OpenShift Container Platform
+                        product=OpenShift Container Platform${product_suffix}
```

Comment:
@cjryan please note that we will be sending a "partial" build notice out on the UMB. Partial builds indicate that some images might not have been built.

Reply:
@jupierce understood, thanks for the heads-up!

Reply:
@jupierce , you mean that we will send only a partial notice or a partial notice in addition to the final notice? We need a way to know when everything is ready.

Reply:
Ok, I understand now what you mean. We need to have this indicated in headers to allow filtering of messages. I think changing message type. Perhaps use message type 'partial_build_done'. Hope such custom message will be allowed by the plugin. I see a `Custom` message in the list. So maybe we can just use `custom` as value? Presently there is an issue with our jenkins and I can't check how it works.

Reply:
@akostadinov Can't you filter on "product"? If not, I will just disable the message on partial builds for now.